### PR TITLE
✨ Fixes #175

### LINKED ISSUE #175: [Frontend] Put a marker/dot on Map for User's location
## Put a marker/dot on Map for User's location 

**Description:**

Currently there is no marker for the user's current geo location. Add a marker similarly it is done for donors and blood banks.

### PR DESCRIPTION
**Pull request by harith-hacky03**
## Description

✨Fixes #175 
Implemented the marker for current user's location

## Related Issue(s)
Fixes #175

## Screenshots (if applicable)
sang

## Checklist

Please review and check the following before submitting your pull request:

- [x] I have followed the [Code of Conduct](CODE_OF_CONDUCT.md).
- [x] My code follows the project's [contributing guidelines](CONTRIBUTING.md).
- [x] I have tested my changes locally and they work as expected.
- [x] I have included relevant tests for my changes.
- [x] I have updated the project documentation if necessary.
- [x] I have reviewed my code to ensure it is free of debugging or development-related code.
- [x] I have provided clear and informative commit messages.

## Contributor License Agreement

By submitting this pull request, I confirm that my contribution is made under the terms of the project's [License](LICENSE.md), and I agree to abide by the project's [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: frontend/src/Components/dashboard/reciever/Home.jsx [hunk 1]
```diff
@@ -392,7 +394,16 @@ const Home = () => {
...
                 style={{ width: 600, height: 340 }}
                 mapStyle="mapbox://styles/mapbox/streets-v9"
+
               >
+                  <Marker 
+                    longitude={longitude}
+                    latitude={latitude}
+                    anchor="bottom"
+                  >
+                  <img src={pin} alt="pin" />
```
Comment (sayakongit) [inline frontend/src/Components/dashboard/reciever/Home.jsx:404]: Could you please add `style={{ width: "26px", height: "40px" }}` to the `img` tag? The default style making it a bit larger.

**Comment by sayakongit** [comment]
@harith-hacky03  Please try to follow [CONTRIBUTING.md](https://github.com/sayakongit/sangnet/blob/master/CONTRIBUTING.md) to maintain the standards in the repo.

**Comment by harith-hacky03** [comment]
@sayakongit Everything is done. Can you check it now?

**Comment by sayakongit** [comment]
LGTM 🥳

**Comment by sayakongit** [comment]
@all-contributor Please add @harith-hacky03 for code

**Comment by sayakongit** [comment]
@all-contributors Please add @harith-hacky03 for code

**Comment by harith-hacky03** [comment]
Thank you so much

**Pull request merged**